proposals/dao/oa_cr_fix.ts

```diff
@@ -0,0 +1,125 @@
+import hre, { ethers, artifacts } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { ZERO_ADDRESS, overwriteChainlinkAggregator } from '@test/helpers';
+
+const fipNumber = 'oa_cr_fix';
+let pcvStatsBefore;
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  // Deploy lens to report B-30FEI-70WETH as WETH and protocol-owned FEI
+  // (The new contract contains a fix)
+  const balancerPool2LensFactory = await ethers.getContractFactory('BalancerPool2Lens');
+  const balancerLensBpt30Fei70WethFixed = await balancerPool2LensFactory.deploy(
+    addresses.gaugeLensBpt30Fei70WethGauge, // address _depositAddress
+    addresses.wethERC20, // address _token
+    '0x90291319f1d4ea3ad4db0dd8fe9e12baf749e845', // IWeightedPool _pool
```

Comment:
`0x90291319f1d4ea3ad4db0dd8fe9e12baf749e845` - should this be put into `mainnetAddresses.ts`?